lib/index.js

```diff
@@ -0,0 +1,322 @@
+// FIXME: should joyent copyright be included here `parseRequest` is mostly
+// copy/paste
```

Comment:
Yes, we should say where it's from and then add our copyright too.

Reply:
Can you provide verbiage?

Reply:
https://github.com/digitalbazaar/forge/blob/master/lib/des.js <-- that's an example of code we pulled in with another BSD copyright.

So, "This implementation was based in part on X. EOL <the copyright from X> EOL <usual DB copyright>"